src/components/utilities/Drop.tsx

```diff
@@ -6,16 +6,17 @@
 
 import React, { HTMLAttributes, useCallback, useState } from 'react';
 
-import styled, { css, SimpleInterpolation } from 'styled-components';
+import styled, { SimpleInterpolation, css } from 'styled-components';
 
-import { Container } from '../layout/Container';
+import { OVERLAY_ELEMENT_IDENTIFIER } from '../constants';
+import { Container, ContainerProps } from '../layout/Container';
 
 const DropEl = styled(Container)`
 	display: inline;
 	position: relative;
 `;
 
-const OverlayEl = styled(Container)`
+const OverlayEl = styled(Container)<ContainerProps & { type: string }>`
```

Comment:
ContainerProps type is already inherited from the Container itself. Re-declaring it is redundant.

I don't like the fact that the type attribute is set to distinguish between elements. I would prefer a data-type instead, which tells more about the fact that that value is a custom attribute.